pandas/core/series.py

```diff
@@ -1789,12 +1789,17 @@ def count(self, level=None):
         """
         if level is None:
             return notna(self.array).sum()
+        elif not isinstance(self.index, MultiIndex):
+            raise ValueError("Series.count level is only valid with a MultiIndex")
```

Comment:
is this hit in a test?

Reply:
no.  ill add one in my just started collected-followups branch